std/format.d

```diff
@@ -3,12 +3,44 @@
 /**
    This module implements the formatting functionality for strings and
    I/O. It's comparable to C99's $(D vsprintf()) and uses a similar
-   format encoding scheme.
+   _format encoding scheme.
 
-   For an introductory look at $(B std.format)'s capabilities and how to use
+   For an introductory look at $(B std._format)'s capabilities and how to use
    this module see the dedicated
    $(LINK2 http://wiki.dlang.org/Defining_custom_print_format_specifiers, DWiki article).
 
+   This module centers around two functions:
+
+$(BOOKTABLE ,
+    $(TR $(TD $(D $(LREF formattedWrite)))
+        $(TD Formats its arguments according to the _format string and puts them
+        to an OutputRange.
+    ))
+    $(TR $(TD $(D $(LREF formattedRead)))
+        $(TD Reads values according to the _format string from an InputRange.
+    ))
+)
+
+   Please see the documentation of function $(D $(LREF formattedWrite)) for a
+   description of the _format string.
+
+   Two functions have been added for convenience:
+
+$(BOOKTABLE ,
+    $(TR $(TD $(D $(LREF _format)))
+        $(TD Returns a newly allocated string with the formatting result.
```

Comment:
It may be worth changing this to "GC-allocated" from "newly allocated" for the sake of clarity.